guidance/src/guidance/guidance_state_machine.cpp

```diff
@@ -60,6 +60,8 @@ namespace guidance
                 if(signal == Signal::ACTIVATED)
                 {
                     current_guidance_state_ = State::ACTIVE;
+                }else if(signal == Signal::INITIALIZED){
```

Comment:
Where is this INITIALIZED coming from? Normally that's from health monitor I believe, is that the same thing that's happening at end of route? If not, maybe it makes more sense to use another signal

Reply:
This initialized comes from Line 122,  at the end of the route the system_alert message is checked, if that is "DRIVERS READY", this INITIALIZED signal is created